mix.exs

```diff
@@ -18,7 +18,7 @@ defmodule Mariaex.Mixfile do
   end
 
   defp deps do
-    []
+    [{:decimal, "~> 1.0"}]
```

Comment:
You also need to add `:decimal` alongside `:logger` a couple lines above as it is a runtime dependency.